src/sentry/lang/javascript/processor.py

```diff
@@ -201,9 +206,9 @@ def discover_sourcemap(result):
                 )
             sourcemap = sourcemap[:index]
         # fix url so its absolute
-        sourcemap = non_standard_url_join(result.url, sourcemap)
+        sourcemap = non_standard_url_join(result.url, force_text(sourcemap))
```

Comment:
This and the following `force_text` calls are probably problematic if the encoding isn't utf-8.